l10n_it_fatturapa_out/models/attachment.py

```diff
@@ -27,6 +27,37 @@ class FatturaPAAttachment(models.Model):
         'res.partner', string='Customer', store=True,
         compute='_compute_invoice_partner_id')
 
+    def get_file_vat(self):
```

Comment:
```suggestion
    @api.model
    def get_file_vat(self):
```
Per rendere chiaro che non dipende dal record corrente